From d2ed05d88c498d26b0447b806a495f79de9bb2f0 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Mon, 11 Mar 2013 20:04:34 -0400
Subject: [PATCH] PlayerDeathEvent.getItemsToKeep

Exposes a mutable array on items a player should keep on death
---
 src/main/java/org/bukkit/event/entity/PlayerDeathEvent.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/main/java/org/bukkit/event/entity/PlayerDeathEvent.java b/src/main/java/org/bukkit/event/entity/PlayerDeathEvent.java
index 1e3198f..69803af 100644
--- a/src/main/java/org/bukkit/event/entity/PlayerDeathEvent.java
+++ b/src/main/java/org/bukkit/event/entity/PlayerDeathEvent.java
@@ -14,6 +14,8 @@ public class PlayerDeathEvent extends EntityDeathEvent {
     private int newLevel = 0;
     private int newTotalExp = 0;
     private boolean keepLevel = false;
+    private List<ItemStack> itemsToKeep = new java.util.ArrayList(); // EMC
+    public List<ItemStack> getItemsToKeep() {return itemsToKeep;} // EMC
 
     public PlayerDeathEvent(final Player player, final List<ItemStack> drops, final int droppedExp, final String deathMessage) {
         this(player, drops, droppedExp, 0, deathMessage);
-- 
1.8.1.5

